Automattic/wp-calypso
# Connections Page: update Like Box Text + Link When Facebook is Connected

Closes https://github.com/Automattic/wp-calypso/issues/83032

## Proposed Changes

This PR replaces the link to `Like Box` with the link to `Page Plugin Widget` on `/marketing/connections/{yourSite}` since `Like Box`[ feature is not available anymore.](https://developers.facebook.com/docs/plugins/like-box-for-pages/)

**Before**

<img width="1051" alt="Screenshot 2024-09-06 at 10 46 23 AM" src="https://github.com/user-attachments/assets/0b7b5e06-4acf-43ff-87d0-121ed849f5dc">

**After**

<img width="1259" alt="Screenshot 2024-09-06 at 10 44 23 AM" src="https://github.com/user-attachments/assets/f17715ba-58e6-4d45-83d2-ddd30e5ce9f6">


## Why are these changes being made?

* `Like Box`[ feature is not available anymore.](https://developers.facebook.com/docs/plugins/like-box-for-pages/) and does not exist.

## Testing Instructions



* Navigate to Calypso Live link: http://calypso.localhost:3000/marketing/connections/{yourSite}
* Click on chevron down icon by Facebook to open the Facebook card
* Open the inspector in your browser tools
* Add the following CSS in your browser inspector:

```
.connections__sharing-service-tip {
    display: block !important;
}
```
* This step is necessary so that you don't have to actually connect a Facebook page to your site
* Observe the `You can also add a [Page Plugin Widget, a share button, or embed your page or profile on your site.` 
* Click on `Page Plugin Widget` link
* Confirm that it navigates to https://wordpress.com/support/facebook-embeds/#facebook-page-plugin-widget

## Pre-merge Checklist
